.drone.yml

```diff
@@ -15,6 +15,27 @@ steps:
   - name: docker
     path: /var/run/docker.sock
 
+- name: fossa
+  image: registry.suse.com/suse/sle15:15.3.17.8.1
+  failure: ignore
+  environment:
+    FOSSA_API_KEY:
+      from_secret: FOSSA_API_KEY
+  commands:
+    - zypper -n install curl unzip
+    - "curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh | sh"
+    - fossa analyze
+    - fossa test
```

Comment:
We should consider building a custom fossa image so that if the curl fails it doesn't kill the run, that would also speed this process up because the image will get cached on runners.

Reply:
One downside of that is if we never update custom fossa image it will never pick up latest sle15 base image(15.3). Not sure if that matters

Reply:
It probably doesn't because its not getting shipped with product. But this change itself is also minimal gain.

Reply:
Well it probably reduces the chance of getting attacked by running YOLO scripts on every build if `https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh` get hacked....

Reply:
@oxr463  @sirredbeard we don't necessarily have to do this, but if we are adding it everywhere then now would be the best time.

Reply:
It's pretty much already in place. I'm just adding some additional repositories.